islandora_ocr.install

```diff
@@ -36,13 +36,19 @@ function islandora_ocr_enable() {
  */
 function islandora_ocr_cleanup_derivatives() {
   if (module_exists('islandora_book')) {
-    $book = variable_get('islandora_book_ingest_derivatives');
+    $book = variable_get('islandora_book_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $book['ocr'] = FALSE;
     variable_set('islandora_book_ingest_derivatives', $book);
   }
 
   if (module_exists('islandora_newspaper')) {
-    $newspaper = variable_get('islandora_newspaper_ingest_derivatives');
+    $newspaper = variable_get('islandora_newspaper_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $newspaper['ocr'] = FALSE;
```

Comment:
This also happens here.  Now I wonder if the issue is different. Do we have something in islandora_newspaper or 'islandora_book' that helps us set those defaults, quick look tells me this is hardcoded in some form options? What if someone changes those defaults or allows a new one?. Will we ever remember we need to change them here too?  So if this piece of code only cares about ocr, would it be not better to default to `variable_get('islandora_newspaper_ingest_derivatives', array('ocr'))` ? Seems safer... thoughts @Islandora/7-x-1-x-committers

Reply:
@robyj sorry this seem to conflict with @whikloj review 😭 . Weird enough we do default to `array(ocr)` in at least some other place... Islandora lacks of consistency sometimes... anyway. Mine are just comments.

Reply:
Hi, I pulled that default value of just array('ocr') from what seemed to be another place it was being set by default. I'd just like it to be correct.

Reply:
@robyj totally. I get it!, I also saw it as only array('ocr') somewhere else (zip importer for newspapers?). The issue is, defaulting here makes almost little sense. Defaulting makes sense when load of variable needs to be consistent with the whole expectation and the logic that comes after. In this case you only want to set ocr to false if the variable is set at all right? So i dare to question, why to even save in case the variable was not set? @whikloj  thoughts? In any case nothing here is life or dead, but still this needs to A) respect what was set, B) reset the OCR to false C) if nothing was there, better leave it like that.

Reply:
Those settings of `array('pdf' => 'pdf', 'image' => 'image', 'ocr' => FALSE)` are just defaults, if the user has ever set their own options that array would be returned instead.

However in this case because we are unsure if we are getting the default or the users setting we have to do the `$newspaper['ocr'] = FALSE;` a second time.

If that makes this too confusing then perhaps just doing a [`drupal_map_assoc(array('ocr', 'image', 'pdf'))`](https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_map_assoc/7.x) would be better. 

That would match how the default is set in the newspaper solution pack. https://github.com/Islandora/islandora_solution_pack_newspaper/blob/7.x/includes/admin.form.inc#L36-L37

Reply:
I should have said that we would then still explicitly set the `$newspaper['ocr'] = FALSE;`

Reply:
Hi, I pulled that default value of just array('ocr') from what seemed to be another place it was being set by default. I'd just like it to be correct.

Reply:
@DiegoPino you're thinking something like this?
```
$newspaper = variable_get('islandora_newspaper_ingest_derivatives', array());
if (isset($newspaper['ocr'])) {
  $newspaper['ocr'] = FALSE;
  variable_set('islandora_newspaper_ingest_derivatives, $newspaper);
}
```

Reply:
Was there any more work for me to do for this?

Reply:
Sorry. This gone slipped under my radar (radar is off-line, forgot to pay the bill)